docs/index.md

```diff
@@ -1,65 +1,68 @@
 # Conftest
 
-Conftest is a utility to help you write tests against structured configuration data. For instance you could write tests for your Kubernetes configurations, or Tekton pipeline definitions, Terraform code, Serverless configs or any other structured data.
+Conftest is a utility to help you write tests against structured configuration data. For instance, you could write tests for your Kubernetes configurations, Tekton pipeline definitions, Terraform code, Serverless configs or any other structured data.
 
 Conftest relies on the Rego language from [Open Policy Agent](https://www.openpolicyagent.org/) for writing the assertions. You can read more about Rego in [How do I write policies](https://www.openpolicyagent.org/docs/how-do-i-write-policies.html) in the Open Policy Agent documentation.
 
 ## Usage
 
-Conftest allows you to write policies using Open Policy Agent/rego and apply them to one or
-more configuration files. Policies by default should be placed in a directory called `policy` but this can be overridden.
+Policies by default should be placed in a directory called `policy`, but this can be overridden with the `--policy` flag.
```

Comment:
Maybe it would be good to introduce the concept of policies a bit more, as above they are referred to as assertions.

Reply:
Good catch, I do prefer the consistency to keep calling them policies. I also updated the link to the documentation site.